business-registry-dashboard/app/components/Form/AddNameRequest/Help.vue

```diff
@@ -11,7 +11,7 @@ defineEmits<{
     <p>{{ $t('form.manageNR.help.description') }}</p>
     <BCRegContactInfo />
     <UButton
-      class="self-start"
+      class="self-start pl-0"
```

Comment:
Are there places we use `self-start` and don't want 0 left padding? If not, can we set 0 left padding in that class itself?

Reply:
Yep, it's there in 6 places with 0 left padding, 3 examples:
https://github.com/bcgov/business-ui/blob/f8ffbaeef81b7bcf62b19e6773111c0276df2391/business-registry-dashboard/app/components/Form/AddBusiness/Error.vue#L107
https://github.com/bcgov/business-ui/blob/f8ffbaeef81b7bcf62b19e6773111c0276df2391/business-registry-dashboard/app/components/Modal/Base.vue#L62
https://github.com/bcgov/business-ui/blob/f8ffbaeef81b7bcf62b19e6773111c0276df2391/business-registry-dashboard/app/components/Modal/Base.vue#L65

Reply:
But are there any places that need the default left padding? If not, can we set 0 left padding in that class itself?

Reply:
> Maybe you're asking if I can change the left padding in that `self-start` class? The `self-start` class in this context is a Tailwind CSS utility class that controls how a flex item positions itself along the cross axis within a flex container. Because of that, it's not possible since it's a Tailwind class.

Yeah, that's what I was thinking.

No, don't change the utility classes.

I'm OK to resolve this conversation.